src/olympia/abuse/cinder.py

```diff
@@ -107,12 +113,21 @@ def id(self):
         return str(self.user.id)
 
     def get_attributes(self):
-        return {
+        data = {
             'id': self.id,
             'name': self.user.display_name,
             'email': self.user.email,
             'fxa_id': self.user.fxa_id,
         }
+        if self.user.picture_type and backup_storage_enabled():
```

Comment:
Oh, just realised, this is going to run each time a `CinderUser` is sent in a request, but we only need the avatar when the `CinderUser` is being reported, not when they're the reporter or a connected user (author, etc)... right?

Reply:
Ah, yes. Maybe I should introduce a `minimal` argument to only include the image when we're gathering attributes for reporting a `CinderUser` ? Or do you see a cleaner way of doing it ?

Reply:
Would a `CinderUser` subclass (e.g. `CinderUserProfile`) make is clearer?  (Would it still be the same entity name in cinder?  I think ...yes?)

Reply:
Yeah that sounds good. Same entity, just more attributes (from my testing they are optional)

Reply:
d53fefd1e68fa1d9d205490c1e348332114784de